doc/languages.rst

```diff
@@ -949,6 +951,19 @@ to view the docstring of the syntax checker.  Likewise, you may use
          A list of additional library directories. Relative paths are relative
          to the buffer being checked.
 
+   .. syntax-checker:: rust-cargo-clippy
```

Comment:
@dzamlo If it's a superset of `rust-cargo` and replaces `rust-cargo` in regular use, shouldn't it use at least the same options as `rust-cargo`?